cluster/ci/config/prow/plugins.yaml

```diff
@@ -1094,3 +1097,19 @@ external_plugins:
     - name: needs-rebase
       events:
       - pull_request
+
+bugzilla:
+  default:
+    openshift-4.1:
+      is_open: true
+      target_release: 4.1.z
+    release-4.1:
+      is_open: true
+      target_release: 4.1.z
+    # TODO: turn this on when we branch
+    # openshift-4.2:
```

Comment:
Let's turn this in now.  Pre-branch PRs should target master and end up in the 4.2 branches via magical ART fast-forwards, so we van block direct-to-4.2 PRs now.

Reply:
There are engineering teams delivering to "master" as openshift-4.2.

Reply:
nope, so these are repos like etcd where they do their dev in this branch. not master